src/systems/exit-on-blur.js

```diff
@@ -2,18 +2,26 @@ AFRAME.registerSystem("exit-on-blur", {
   init() {
     this.onBlur = this.onBlur.bind(this);
     this.onFocus = this.onFocus.bind(this);
+    this.onTimeout = this.onTimeout.bind(this);
+
+    this.isOculusBrowser = navigator.userAgent.match(/Oculus/);
 
     window.addEventListener("blur", this.onBlur);
     window.addEventListener("focus", this.onFocus);
 
     this.exitTimeout = null;
   },
 
+  tick() {
+    if (this.isOculusBrowser) {
```

Comment:
I would add a note about what this hack is for so that we know when it can be removed (when the WebVR events get fixed)